setup.cfg

```
@@ -38,7 +38,7 @@ setup_requires =
 install_requires =
     entrypoints >=0.2.2
     jupyterlab ~=3.0
-    jupyterlab-git >=0.30.0,<0.40.0a0
+    jupyterlab-git >=0.30.0,<=0.41.0
```

Good catch - let's even be more tolerant (I'm finalizing the integration of some new features in JupyterLab Git extension).

```suggestion
    jupyterlab-git >=0.30.0,<0.50.0
```